docs_src/microservices/device/supported/device-onvif-camera/Walkthrough/custom-build.md

```diff
@@ -29,6 +29,12 @@ Configuring pre-defined devices will allow the service to automatically provisio
    cp ./cmd/res/devices/camera.yaml.example ./cmd/res/devices/camera.yaml
    ```
 
+      <div class='admonition warning'>
+         <p class='admonition-title'>Warning</p>
+         <p>Be careful when storing any potentially important information in cleartext on files in your computer. Potentially sensitive information in this case could include the IP address of your ONVIF camera or any custom metadata you configure.</p>
+      </div>
```

Comment:
Please use the mkdocs warning here any other location where you have this embedded HTML
```suggestion
!!! Caution
    Be careful when storing any potentially important information in cleartext on files in your computer. Potentially sensitive information in this case could include the IP address of your ONVIF camera or any custom metadata you configure.
```

Reply:
It is all about proper indentation. Mkdocs requires 4 spaces of indentation including the in-line bash block above. Once I aligned them it renders as expected.

Reply:
![image](https://user-images.githubusercontent.com/44779287/235742677-f9de2105-71d4-4d3b-b57e-aaa41d6dab79.png)